### PR TITLE
update CONTRIBUTING.md to the latest template standard

### PR DESCRIPTION
**Pull request by soerenmartius**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,75 +2,92 @@
...
 
-The [CODEOWNERS](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners)
-of the repository will review your code and provide feedback. If everything looks good, they will merge the code and
-release a new version while following the principles of [Semantic Versioning (SemVer)](https://semver.org/).
+The [CODEOWNERS] of the repository will review your code and provide feedback.
+If everything looks good, they will merge the code and release a new version while following the principles of [Semantic Versioning (SemVer)].
+
+<!-- References -->
+
+<!-- markdown-link-check-disable -->
```
Comment (mariux) [inline CONTRIBUTING.md:80]: all links should work as repo is public

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,75 +2,92 @@
...
+[Pull Requests]: https://github.com/mineiros-io/terraform-module-template/pulls
```
[the remark this answers is not in the document]
Comment (soerenmartius) [reply CONTRIBUTING.md]: resolved in 0de9238

File: CONTRIBUTING.md (hunk 1 above)
Comment (soerenmartius) [reply CONTRIBUTING.md]: resolved in 0de9238

**Comment by mariux** [comment]
[ch602]

**Pull request merged**